LICENSE.md

```diff
@@ -70,6 +70,7 @@ The following components of Julia's standard library have separate licenses:
 Julia builds the following libraries by default, but does not use them itself:
 
 - [RMATH](http://www.r-project.org/Licenses/)
+- [LDC](https://github.com/ldc-developers/ldc/blob/master/LICENSE)
```

Comment:
> i'm not really sure where this should go however. none of the existing sections really apply to the idea that we just took part of their code, but don't actually use any part of LDC.

I'd create a new section just below `The following components of Julia's standard library have separate licenses:`. Also, it would be nice to give more precisions, as the linked file gives several different licenses depending on the parts of the code. If the exceptions do not apply to the parts that were adapted, it would make the interpretation of the Julia license as a whole much easier.